e2e/packages/wp-source-errors/src/index.tsx

```diff
@@ -28,8 +28,10 @@ const wpSourceErrors: WpSourceErrors = {
           state.source.api = "https://test.frontity.io/wp-json";
         }
       },
-      beforeSSR: async ({ state, actions }) => {
+      beforeSSR: ({ state, actions }) => async ctx => {
```

Comment:
@luisherranz This was not relevant for my tests, because I needed to rely on the `beforeSSR` of the `tiny-router`. 

The strange thing is that with your changes, I can comment out the content of the `beforeSSR` in this file and the tests pass **and** I get the correct errors when I navigate to the page manually with my browser.

Reply:
Oh, do you mean that the tests pass without any code? Maybe the tests are wrong... I don't know. Could you please check them out?

Reply:
I meant that they pass because of this: https://github.com/frontity/frontity/pull/309/files#diff-d8058b9e7fa71c069f1e5c2f6a255444L50-R65

We set the correct status code in the `beforeSSR` action of the `tiny-router`.